src/Logic/OCR/OcrFixEngine.cs

```diff
@@ -1350,21 +1350,8 @@ public string FixUnknownWordsViaGuessOrPrompt(out int wordsNotFound, string line
 
                     if (!correct && word.Contains('/') && !word.Contains("//"))
                     {
-                        var slashedWords = word.Split(new[] { '/' }, StringSplitOptions.RemoveEmptyEntries);
-                        bool allSlashedCorrect = true;
-                        foreach (var slashedWord in slashedWords)
-                        {
-                            if (slashedWord.Length < 2)
-                            {
-                                allSlashedCorrect = false;
-                            }
-
-                            if (allSlashedCorrect && !(DoSpell(slashedWord) || IsWordKnownOrNumber(slashedWord, line)))
-                            {
-                                allSlashedCorrect = false;

```

Comment:
No `break` after the allSlashedCorrect changes state (the state will never change again inside loop)